idtoken/validate_test.go

```diff
@@ -231,6 +231,42 @@ func TestValidateES256(t *testing.T) {
 	}
 }
 
+func TestGetPayload(t *testing.T) {
+	idToken, _ := createRS256JWT(t)
+	tests := []struct {
+		name                string
+		token               string
+		wantPayloadAudience string
+		wantPayload         *Payload
```

Comment:
```suggestion
		wantPayloadAudience string
```
`wantPayload` isn't used anywhere in the test or cases.